src/changelogs/tags.py

```diff
@@ -0,0 +1,342 @@
+            'minimaps': 'Map',
+            'Minimap': 'Map',
+            'parry': 'Parry',
+            'parried': 'Parry',
+            'light melee': 'Light Melee',
+            'heavy melee': 'Heavy Melee',
+            'Light melee': 'Light Melee',
+            'Heavy melee': 'Heavy Melee',
+            'melee': 'Melee',
+            'melees': 'Melee',
+            'pause': 'Pause',
+            'pauses': 'Pause',
+            'paused': 'Pause',
+            'vaults': "Sinner's Sacrifice",
+            'vault': "Sinner's Sacrifice",
+            'Vault': "Sinner's Sacrifice",
+            "Sinner's sacrifice": "Sinner's Sacrifice",
+            'Sinners sacrifice': "Sinner's Sacrifice",
+            "Sinners' sacrifice": "Sinner's Sacrifice",
+            'Sinner sacrifice': "Sinner's Sacrifice",
+            "sinner's sacrifice": "Sinner's Sacrifice",
+            'sinners sacrifice': "Sinner's Sacrifice",
+            "sinners' sacrifice": "Sinner's Sacrifice",
+            'sinner sacrifice': "Sinner's Sacrifice",
+            'flex slot': 'Flex Slot',
+            'Flex slot': 'Flex Slot',
+            'flex slots': 'Flex Slot',
+            'Flex': 'Flex Slot',
+            'guns': 'Gun',
+            'gun': 'Gun',
+            'Guns': 'Gun',
+            'Pits': 'Pit',
+            'pit': 'Pit',
+            'pits': 'Pit',
+        }
+
+        # Relations between a child and parent tag.
+        # i.e. Abrams is a parent to Siphon Life, and a child to Hero
+        # tags below are after _remap_tag() is called
+        # key = parent
+        # value = children
+        # also see self.parent_lookup
+        self.tag_tree = {
+            'Item': {'Weapon Item': {}, 'Vitality Item': {}, 'Spirit Item': {}},
+            'Hero': {'Ability': {}, 'Gun': {}},
+            'Melee': {
+                'Light Melee': {},
+                'Heavy Melee': {},
+                'Parry': {},
+            },
+            'NPC': {'Denizen': {}, 'Creep': {'Denizen': {}, 'Trooper': {}}, 'Mid-Boss': {}},
+            'Objective': {
+                'Guardian': {},
+                'Base Guardian': {},
+                'Walker': {},
+                'Patron': {'Weakened Patron': {}},
+                'Shrine': {},
+            },
+            'Souls': {'Soul Orb': {}, "Sinner's Sacrifice": {}, 'Soul Urn': {}},
+            'Breakable': {'Crate': {}, 'Golden Statue': {}},
+            'Shop': {},
+            'Map': {
+                'Department Store': {},
+                'Arcade': {},
+                'Fish Market': {},
+                'Basement': {},
+                'Bridge': {},
+                'Hotel': {},
+                'Pharmacy': {},
+                'Penn Station': {},
+                'Grand Central': {},
+                'Pit': {},
+            },
+            'Powerup': {},
+            'Zipline': {},
+            'Rejuvenator': {},
+            'Cosmic Veil': {},
+            'Sapphire Hand': {},
+            'Amber Hand': {},
```

Comment:
Perhaps these fit inside a `Lore` group? idk enough about what those are

Reply:
I thought about this too, though looking at where they are used, it is very generic. i.e.
"* {{PageRef|Sapphire Hand|alt_name=Sapphire}} side materials tint color changed to white"

Here it's just being used to represent the team. My original idea was to nest them under something more generic like "Team", but it feels too generic to warrant it's existence. As I'm writing this I realize it does warrant it's existence lol. Moving em under "Team" tag.

Reply:
Oh yeah those are the names of the teams 🤦. Yeah no harm in having it